kyuubi-server/src/main/scala/org/apache/kyuubi/session/KyuubiSessionManager.scala

```diff
@@ -361,13 +368,29 @@ class KyuubiSessionManager private (name: String) extends SessionManager(name) {
     batchLimiter.foreach(SessionLimiter.resetUnlimitedUsers(_, unlimitedUsers))
   }
 
+  private[kyuubi] def getLimitedUsers: Set[String] = {
+    limiter.orElse(batchLimiter).map(SessionLimiter.getLimitedUsers).getOrElse(Set.empty)
+  }
+
+  private[kyuubi] def refreshLimitedUsers(conf: KyuubiConf): Unit = {
+    val limitedUsers = conf.get(SERVER_LIMIT_CONNECTIONS_USER_LIMITED_LIST).toSet
```

Comment:
The user list should be filtered ensuring non-empty elements.

Reply:
done.